blocks/gdplus-subscribe-hero/gdplus-subscribe-hero.js

```diff
@@ -0,0 +1,149 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { parseFragment, removeEmptyElements, render } from '../../scripts/scripts.js';
+
+const offerCards = [
+  {
+    title: 'Digital Only',
+    cost: '$1.99',
+    scope: 'mo',
+    billScope: 'annually',
+  },
+  {
+    title: 'Digital & Print',
+    cost: '$39.99',
+    scope: 'yr',
+    billScope: 'annually',
+    mostPopular: true,
+  },
+  {
+    title: 'Digital + Print + Golf Digest Schools',
+    cost: '$169.99',
+    scope: 'yr',
+    billScope: 'annually',
+  },
+];
+
+const benefits = [
+  {
+    title: 'Unlimited GolfDigest.com articles',
+  },
+  {
+    title: 'Digital subscription to Golf Digest magazine',
+  },
+  {
+    title: "Unlimited access to Golf Digest's 17,000+ course reviews",
+  },
+  {
+    title: 'Exclusive digital features, including interactive experiences',
+  },
+  {
+    title: 'Curated newsletters',
+  },
+  {
+    title: 'Print subscription to Golf Digest Schools',
+    info: 'Included only in the Digital + Print or Digital + Print + GD Schools plans',
+  },
+  {
+    title: 'Unlimited access to Golf Digest Schools',
+    info: 'Included only in the Digital + Print + Golf Digest Schools plans',
+  },
+  {
+    title: 'Unlimited access to the Golf Digest Archive - since 1950',
+  },
+];
+
+const HTML_TEMPLATE = `
+<div class="gd-plus-subscribe-content">
+  <div class="background-image-wrapper">
+    <div class="image-overlay"></div>
+    ${createOptimizedPicture('https://main--helix-sportsmagazine--headwirecom.hlx.live/subscribe-background-mockup.jpg', 'hero background green', true, [
+    { width: '1690' },
```

Comment:
Can you try adding other sizes for mobile too ? Showing the 1690px width image on mobile is probably not optimal performance wise. 

<img width="578" alt="Screenshot 2023-09-05 at 13 06 33" src="https://github.com/headwirecom/helix-sportsmagazine/assets/6012307/5d9736c5-9f73-41a5-9ec3-fe5734d9a3d0">

Reply:
@icaraps I added some breakpoints, and performance is now over 90 (inconsistent) but the image gets very blurry on mobile. The way the page is styled & the aspect ratio of the image, the intrinsic width doesn't really go below 1000px.

Reply:
@DriftingSands  I see, i think it's fine you don't really see it anyway on mobile since it's almost fully covered. 90 is ok on .page, it means on .live aka production, you'll get more so you are consistently passing the check.